test/system/aws/preTestScript.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+aws_access_key=${1:-null}
+aws_secret_key=${2:-null}
+aws_region=${3:-null}
+aws_key_name=${4:-aws-key-pair}
+cred_path=${5:-null}
+config_path=${6:-null}
+pravega_org=${7:-pravega}
+pravega_branch=${8:-master}
+cd aws/
+terraform init
```

Comment:
Couple of questions related to terraform configuration.
1. Is the cluster destroyed conditionally once the test execution is completed?
2. How does log copy work? Do we use TF to perform it or should we have `postTestScript.sh`

Reply:
no , 1 & 2 are not done in the PR , will do that as part of travis job issue.